src/app/util/types_stub.h

```diff
@@ -46,6 +46,20 @@
 
 #include <app/chip-zcl-zpro-codec.h> // For EmberApsFrame
 
+/**
```

Comment:
Maybe we should have a NodeId header that declares them together to make it obvious they are both uint64_t.

Reply:
Filed https://github.com/project-chip/connectedhomeip/issues/3316